examples/PatternSelector-wizard-with-packages.py

```diff
@@ -22,8 +22,7 @@ def main(self):
 
       UI.OpenDialog(
         Opt("defaultsize"),
-        #Wizard("back", "", "cancel", "&Cancel", "ok", "&OK")
-        Wizard(Opt("stepsEnabled"), Id("back"), "", Id("cancel"), "&Cancel", Id("ok"), "&OK")
+        Wizard(Symbol("back"), "", Symbol("cancel"), "&Cancel", Symbol("ok"), "&OK")
```

Comment:
Why did you remove `Opt("stepsEnabled")`?

Reply:
because it isn't in the orig Ruby example in ycp-ui-bindings, see the comment I also removed, I think this was an attempt to fix the Symbol/String ambiguity (in the initial work)